data/projects.json

```diff
@@ -228,5 +228,15 @@
         "level": "intermediate",
         "site": "https://brave-mirzakhani-8081ab.netlify.app/",
         "tags": "vuejs, pico css, netlify"
+    },
+    {
+        "name": "codeet",
```

Comment:
Please check the repo's name or make the repo as public
<img width="883" alt="image" src="https://user-images.githubusercontent.com/37890026/160124911-4f5e9d0b-ffe6-49f0-a3f7-214dc239266a.png">